front/app/components/IdeaCard/index.tsx

```diff
@@ -187,24 +199,43 @@ const CompactIdeaCard = memo<IdeaCardProps>(
       return null;
     };
 
-    const onCardClick = (event: FormEvent) => {
-      event.preventDefault();
+    useEffect(() => {
+      if (scrollToCardParam && idea.data.id === scrollToCardParam) {
+        const subscription = eventEmitter
+          .observeEvent(IMAGES_LOADED_EVENT)
```

Comment:
This is the only way we can know that the images have fully loaded the moment we scroll to the correct element. Without this, you scroll before the images are loaded, and then when they load the scroll position in messed up.